components/src/core/components/DropdownMenu/dropdown-menu.scss

```diff
@@ -15,6 +15,29 @@
   border: $oxd-dropdown-border;
   border-radius: 6px;
   box-shadow: 0px 12px 20px 0px rgb(0 0 0 / 8%);
+
+  &.--positon-top {
```

Comment:
Get these into a single CSS and import them.

Reply:
1. Moved "$oxd-dropdown-dropdown-margin: 5px !default;" to root variables.scss
2. Defined "--positon-top, bottom, left and right" utility classes in root utility.scss
3. Extended the above utilities inside dropdown-menu.scss using scss @extend